inst/@sym/horzcat.m

```diff
@@ -1,4 +1,4 @@
-%% Copyright (C) 2014-2017, 2019 Colin B. Macdonald
+%% Copyright (C) 2022 Alex Vong
```

Comment:
suggest leaving my copyright notice here, unless its a brand new file.

Reply:
Aha, this is a typo. I was suppose to a copy & edit.

Reply:
I also forget to add copyright notice in `inst/@sym/transpose.m`